src/main/java/org/broadinstitute/hellbender/engine/dataflow/datasources/RefAPISource.java

```diff
@@ -277,6 +293,8 @@ private Genomics createGenomicsService(final PipelineOptions pipelineOptions) {
         }
     }
 
+    // TODO: Move these to a CustomCoder. That will allow us to do something else (possibly better) for Spark.
+    // TODO: See Issue #849.
```

Comment:
Is this difficult to do now? If not, I'd recommend doing it as part of this PR so it doesn't get forgotten.